packages/mobile/src/features/videos/components/MainScreen/MainScreenHeader.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import styled from 'styled-components';
+import SmallHeaderButton from './SmallHeaderButton';
+import { NavigationName } from '../../../../config/routing/NavigationName';
+import BigHeaderButton from './BigHeaderButton';
+
+const Wrapper = styled.ScrollView`
```

Comment:
jak puste to usuń i uzyj poprostu ScrolView

Reply:
ok